include/SDL3/SDL_audio.h

```diff
@@ -1116,10 +1113,9 @@ extern DECLSPEC SDL_AudioStream *SDLCALL SDL_CreateAndBindAudioStream(SDL_AudioD
  *
  * \param src The data source for the WAVE data
  * \param freesrc If non-zero, SDL will _always_ free the data source
- * \param spec A pointer to an SDL_AudioSpec that will be set to the WAVE
- *             data's format details on successful return.
+ * \param spec if SDL_TRUE, calls SDL_RWclose() on `src` before returning, even in the case of an error
  * \param audio_buf A pointer filled with the audio data, allocated by the
- *                  function.
+ *                  function
```

Comment:
The documentation changes are clearly wrong: AFAICS, only the freesrc needs changing from `non-zero` to `SDL_TRUE`, yes?

Reply:
Indeed, I modified the wrong line.